### ISSUE TITLE
Create Site Brand/Logo

### ISSUE THREAD

**Issue by ThomasKDesign** [suwebdev/sparkling-art#20]
Authored by: Thomas Kaiaua
URL of the requirements: https://github.com/suwebdev/sparkling-art/wiki/User-Story:-View-Public-Website
Size of the Requirement: 2

**Your reasoning for adding this Issue to the Sprint:** 
At this stage in the project, I figure we should at least have picked out a logo for our site.
**What is the priority that has been assigned?:** 
Priority can vary, probably 2 to 3, as while it's important to have a logo, we still need to leave room for change should we agree to change it before deployment.
**How do you interpret this fitting into the process of building the project? :** 
With every product and project, there has to be a unique identity to it that helps the user to remember and recognize. So it is a must-have feature to the project!
**Why should this be in the Sprint this time? Why not wait until the next Sprint?:** 
The content and placement of them is coming together, and having a brand logo to our site now will allow us some time before to tweak it, per collaborator's suggestion.

**Comment by ThomasKDesign** [comment]
Before any of you guys close/approve this, can you confirm the size? I put 2 for this, but on the issue template I put down 7 thinking that's for the total of the whole issue combining mine, Afi's and Marlie's?? Feel free to correct it if I got it wrong!

**Comment by crossm2** [comment]
I agree this would be a 1 or a 2. Creating the favicon probably took a little design work and then the effort to including it in the code was minimal, so I would go with 2! But yes, for the total combine issue, it was more like a 5-7.
